.github/workflows/release.yml

```diff
@@ -0,0 +1,14 @@
+name: CI build
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set up node
+        uses: actions/setup-node@v1
+      - name: Install dependencies with yarn
+        run: npm install
+#      - name: Test
```

Comment:
Wrong indentations

Reply:
it was a commented line.. but removed it now.